src/memory.rs

```diff
@@ -524,44 +538,44 @@ const BLOCK_SIZE: usize = 64;
 #[derive(Clone, Debug)]
 pub struct UndefMask {
```

Comment:
That sounds good to me. I agree `Size` makes things simpler in some places, but not here.